src/main/java/com/jm/lms/studentms/repository/TaskRepository.java

```diff
@@ -0,0 +1,22 @@
+package com.jm.lms.studentms.repository;
+
+import java.util.Optional;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import com.jm.lms.studentms.model.Task;
+import java.util.List;
+
+
+@Repository
+public interface TaskRepository extends JpaRepository<Task, Long> {
```

Comment:
Change repository name to ViewTaskRepository

Reply:
Done Sir